cityofaustin/atd-moped
# Adds Smart Mobility tags and updates a tag for Arterial Management

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13268
- https://github.com/cityofaustin/atd-data-tech/issues/13398

## Testing
**URL to test:** Local

**Steps to test:**
1. 🔑 Start your local Moped 
2. 🕵️ Inspect the tag options on any project's summary tab. Observe the three tags of concern:
- `Smart Mobility - Expression of Interest`
- `Smart Mobility - Pilot`
- `Arterial Management`

You can also peep the data dictionary to accomplish the same task 😄 

---
#### Ship list
